docs/index.rst

```diff
@@ -50,7 +53,7 @@ Projects that use Nox
 
 Nox is lucky to have several wonderful projects that use it and provide feedback and contributions.
 
-- `Bezier <https://github.com/dhermes/bezier>`__
+- B |eacute| zier `project <https://github.com/dhermes/bezier>`__
```

Comment:
This change was unrelated to the PR, I can revert if desired. After doing this it's the **only** bullet that has some non-link text. (I was doing this to avoid non-ASCII in this file, but I now realize the file already has non-ASCII so I'll ditch the `eacute`.)